server/services/manageUsersService.ts

```diff
@@ -7,7 +7,7 @@ export default class ManageUsersService {
     if (username) {
       try {
         return await this.manageUsersApiClient.users.get(token, { username })
-      } catch (e) {
+      } catch {
```

Comment:
As upgraded Lint claim about unused 'e' variable

Reply:
what about logging the exception instead?